### PR TITLE
DP-25201: Modify service page migration logic for integration branch

### PR DESCRIPTION
**Pull request by arthurbaghdas**
**Description:**
Explain the technical implementation of the work done.

**Jira:** (Skip unless you are MA staff)
DP-25201

**To Test:**
- [ ] Add steps to test this feature

---

[Peer Review Checklist](https://github.com/massgov/openmass/blob/develop/docs/peer_review_checklist.md)

### CODE REVIEW

File: docroot/modules/custom/mass_content/includes/mass_content.service_page.inc [hunk 1]
```diff
@@ -11,7 +11,7 @@ use Drupal\mayflower\Helper;
 /**
  * Helper function to check if paragraph of given type is already populated.
  */
-function check_paragraph($node, $bundle) {
+function check_paragraph_bundle($node, $bundle, $heading = NULL) {
```
Comment (tom-fleming) [inline docroot/modules/custom/mass_content/includes/mass_content.service_page.inc]: @arthurbaghdas It doesn't look like the `$heading` argument is used here. Is it needed?

**Comment by tom-fleming** [comment]
@arthurbaghdas Not sure why this wasn't called out or caught before, but the drush command is calling this currently non-static method as a static method, which triggered an error: https://github.com/massgov/openmass/blob/DP-25201_modify_service_page_migration/docroot/modules/custom/mass_content/src/MassContentBatchManager.php#L88

File: docroot/modules/custom/mass_content/includes/mass_content.service_page.inc (hunk 1 above)
Comment (arthurbaghdas) [reply docroot/modules/custom/mass_content/includes/mass_content.service_page.inc]: Forgot to remove it, I was testing another approach

**Comment by tom-fleming** [comment]
LGTM!

**Pull request merged**